src/components/Kv/KvMap.vue

```diff
@@ -0,0 +1,297 @@
+<template>
+	<div class="tw-relative tw-block tw-w-full" :style="mapDimensions">
+		<div
+			:id="`kv-map-holder-${this.mapId}`"
+			:ref="refString"
+			class="tw-w-full tw-h-full tw-bg-black"
+		></div>
+	</div>
+</template>
+
+<script>
+import { createIntersectionObserver } from '@/util/observerUtils';
+import testDelayedGlobalLibrary from '@/util/timeoutUtils';
+
+export default {
+	metaInfo() {
+		return {
+			script: [].concat(!this.hasWebGL ? [
+				// leaflet - uses raster tiles for additional browser coverage
+				{
+					vmid: `leafletjs${this.mapId}`,
+					src: 'https://unpkg.com/leaflet@1.7.1/dist/leaflet.js',
+					async: true,
+					defer: true,
+				},
+			] : []).concat(this.hasWebGL ? [
+				// maplibregl - uses vector tiles and webgl for rendering
+				{
+					vmid: `maplibregljs${this.mapId}`,
+					src: 'https://cdn.maptiler.com/maplibre-gl-js/v1.14.0/maplibre-gl.js',
+					async: true,
+					defer: true,
+				},
+			] : []),
+			link: [
+			].concat(!this.hasWebGL ? [
+				// leaflet - uses raster tiles for additional browser coverage
+				{
+					vmid: `leafletcss${this.mapId}`,
+					rel: 'stylesheet',
+					href: 'https://unpkg.com/leaflet@1.7.1/dist/leaflet.css'
+				},
+			] : []).concat(this.hasWebGL ? [
+				// maplibregl - uses vector tiles and webgl for rendering
+				{
+					vmid: `maplibreglcss${this.mapId}`,
+					rel: 'stylesheet',
+					href: 'https://cdn.maptiler.com/maplibre-gl-js/v1.14.0/maplibre-gl.css'
+				},
+			] : []),
+		};
+	},
+	props: {
+		/**
+		 * Aspect Ration for computed map dimensions
+		 * We'll divide the container width by this to determine the height
+		 */
+		aspectRatio: {
+			type: Number,
+			default: 1.3
+		},
+		/**
+		 * Control how quickly the autoZoom occurs
+		 */
+		autoZoomDelay: {
+			type: Number,
+			default: 1500
+		},
+		/**
+		 * These maps require a height to be set in order to render
+		 */
+		height: {
```

Comment:
Is this prop still needed?

Reply:
It's not currently used. I may add some logic to override the aspectRatio driven values if the height and/or width are provided...

Reply:
Added width and height overrides in latest update.